service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/BlobCopierConnectedTest.java

```diff
@@ -43,32 +50,43 @@ public class BlobCopierConnectedTest extends BaseAzureConnectedTest {
   @Autowired private WorkspaceService workspaceService;
   @Autowired private JobService jobService;
   @Autowired private UserAccessUtils userAccessUtils;
+  @Autowired private LandingZoneTestUtils landingZoneTestUtils;
+  private TestLandingZoneManager testLandingZoneManager;
+  @Autowired private AzureCloudContextService azureCloudContextService;
+  @Autowired private LandingZoneDao landingZoneDao;
+  @Autowired private WorkspaceDao workspaceDao;
+  @Autowired private CrlService crlService;
+  @Autowired private AzureConfiguration azureConfig;
 
   private ControlledAzureStorageContainerResource sourceContainer;
   private ControlledAzureStorageContainerResource destContainer;
-  private ControlledAzureStorageResource storageAcct;
+  private StorageAccount storageAcct;
   private AuthenticatedUserRequest userRequest;
   private UUID workspaceId;
 
   @BeforeAll
   void setup() throws InterruptedException {
-    userRequest = userAccessUtils.defaultUserAuthRequest();
-    Workspace workspace = createWorkspaceWithCloudContext(workspaceService, userRequest);
+    Workspace workspace =
+        createWorkspaceWithCloudContext(workspaceService, userAccessUtils.defaultUserAuthRequest());
     workspaceId = workspace.getWorkspaceId();
 
-    var storageAccountId = UUID.randomUUID();
-    var saName = generateAzureResourceName("sa");
+    // create quasi landing zone with a single resource - shared storage account
+    var storageAccountName = String.format("lzsharedstacc%s", TestUtils.getRandomString(6));
+    UUID landingZoneId = UUID.fromString(landingZoneTestUtils.getDefaultLandingZoneId());
+    testLandingZoneManager =
+        new TestLandingZoneManager(
+            azureCloudContextService,
+            landingZoneDao,
+            workspaceDao,
+            crlService,
+            azureConfig,
+            workspace.getWorkspaceId());
+
     storageAcct =
-        ControlledAzureStorageResource.builder()
-            .common(
-                ControlledResourceFixtures.makeDefaultControlledResourceFieldsBuilder()
-                    .workspaceUuid(workspaceId)
-                    .resourceId(storageAccountId)
-                    .region("eastus")
-                    .build())
-            .storageAccountName(saName)
-            .build();
-    createResource(workspaceId, userRequest, storageAcct);
+        testLandingZoneManager.createLandingZoneWithSharedStorageAccount(
+            landingZoneId, workspaceId, storageAccountName, "eastus");
```

Comment:
Perhaps this should be changed to use the `Region` class, similar to how you did in other tests (`Region.US_EAST2.name()` for example).